src/main/java/org/jetbrains/bsp/bazel/server/bsp/resolvers/TargetsLanguageOptionsResolver.java

```diff
@@ -14,17 +14,18 @@
 import org.jetbrains.bsp.bazel.server.bazel.data.BazelProcessResult;
 import org.jetbrains.bsp.bazel.server.bazel.params.BazelRunnerFlag;
 
-public class TargetsResolver<T> {
+public class TargetsLanguageOptionsResolver<T> {
```

Comment:
it's a bit of a mouthful, I think I prefer the previous name

Reply:
previous name was too general for sure -- now I introduced another targetsResolver